test/crypto.t.mjs

```diff
@@ -0,0 +1,527 @@
+/*---
+includes: [runTsuite.js]
+flags: [async]
+---*/
+
+import cr from 'crypto';
+
+
+function p(args, default_opts) {
+    let params = merge({}, default_opts);
+    params = merge(params, args);
+
+    return params;
+}
+
+
+let createHash_tsuite = {
```

Comment:
Tests can be made more regular and aligned with other existing tests. I will share the patch.